src/App.js

```diff
@@ -28,22 +29,17 @@ const alex = {
 
 const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">My wife&apos;s name is Natasha</p>
-    </section>
+    <Person
+      person={misha}
+    />
 
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">My husband&apos;s name is Maksym</p>
-    </section>
+    <Person
+      person={olya}
+    />
```

Comment:
[CODE STYLE] - If you have < 3 attributes on a tag and the values are short write the tag in one line (to make it easier to write and read).